stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequestV2.kt

```diff
@@ -165,5 +173,46 @@ class AnalyticsRequestV2(
         internal const val PARAM_EVENT_ID = "event_id"
 
         private const val INDENTATION = "  "
+
+        fun create(
+            eventName: String,
+            clientId: String,
+            origin: String,
+            params: Map<String, *>,
+        ): AnalyticsRequestV2 {
+            return AnalyticsRequestV2(
+                eventName = eventName,
+                clientId = clientId,
+                origin = origin,
+                params = params.toJsonElement(),
+            )
+        }
+    }
+}
+
+private fun List<*>.toJsonElement(): JsonElement {
```

Comment:
This is not yet used right? should we create another `create` with `params: List` ?

Reply:
Also if you add the `params: List`, would be great to add a corresponding test in `AnalyticsRequestVeFactoryTest`